### PR TITLE
XW-2 add quotes if needed to scss values

### PR DESCRIPTION
**Pull request by bognix**
/cc @macbre @hakubo @vforge

### CODE REVIEW

File: includes/wikia/services/sass/Compiler/LibSassCompiler.class.php [hunk 1]
```diff
@@ -98,18 +96,25 @@ public function compile( Source $source ) {
...
 
-		return sprintf("(%s)", join(', ', $pairs));
+		return sprintf( "(%s)", join( ', ', $pairs ) );
+	}
+
+	private function quoteIfNeeded( $item, $key ) {
+		if ( !is_numeric( $item ) && strpos( $key, 'color' ) === false ) {
+			$item = "'{$item}'";
+		}
+		return $item;
```
Comment (macbre) [inline includes/wikia/services/sass/Compiler/LibSassCompiler.class.php:118]: Would be great to add a unit test with handful of cases

File: includes/wikia/services/sass/Compiler/LibSassCompiler.class.php [hunk 2]
```diff
@@ -98,18 +96,25 @@ public function compile( Source $source ) {
...
 
-			$pairs[] = sprintf('"%s": %s', $key, $value);
+			$pairs[] = sprintf( '"%s": %s', $key, $value );
 		}
 
-		return sprintf("(%s)", join(', ', $pairs));
+		return sprintf( "(%s)", join( ', ', $pairs ) );
+	}
+
+	private function quoteIfNeeded( $item, $key ) {
```
Comment (macbre) [inline includes/wikia/services/sass/Compiler/LibSassCompiler.class.php]: PHP docs with a short description would be great

**Comment by vforge** [comment]
LGTM

**Comment by hakubo** [comment]
LGTM

**Comment by macbre** [comment]
LGTM

**Pull request merged**